.github/workflows/bump-opbeans-ruby.yml

```diff
@@ -5,6 +5,7 @@
 name: bump-opbeans-ruby
 
 on:
+  workflow_dispatch: ~
```

Comment:
OOC, what's the reason for `~`? I could not find any references in the [docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#on)

Reply:
It's YAML syntax and it's just a symbol for `null`

IMO makes it explicit that its `null`

instead of 

```yaml
on:
 workflow_dispatch:
```

Reply:
http://yaml.org/spec/1.2-old/spec.html#id2805071

Reply:
Thanks Jan!